### ISSUE TITLE
The Suggester: [SIMPLE REQUEST]: Add a 3-can version of Pepper Spray

### ISSUE THREAD

**Issue by TheSuggester** [eezstreet/SWATEliteForce#417]
If we can have 3 door wedges and 3 grenades as a option, why can't we have a 3-can version of the Pepper Spray?

That might come in handy in QMM (Quick Mission Maker) missions...

**Comment by LeRutY** [comment]
Yeah with many civs (because everyone who punches civs other than sus should get *CENSORED*) and they can be take two pepperspray shots, cans can become empty really quick.

**Comment by T-MaxWiese-T** [comment]
I think that the menu for the tactical things should be the same as for the weapons, because it takes a very long time to click from right to left and also that you can't enter a number like with ammunition I find very complicated.
